Makefile

```diff
@@ -9,6 +9,10 @@ PREFIX?=/usr/local
 CPPFLAGS += -Isrc/ -I$(DESTDIR)$(PREFIX)/include/
 CFLAGS += -O2 -Wall -Werror
 
+# The sljit commit (hashtag or branch) that is downloaded and installed
+# by the make target: install-sljit
```

Comment:
```suggestion
# The sljit commit (hash, branch or tag) that is downloaded and installed
# by the make target: install-sljit
```

I don't think "hashtag" is git terminology. Just "hash". (In git's manpages I find many words for this: "revision", "commit", "hash", "object" and "40-byte hexadecimal commit object name".)

Maybe also worth mentioning that the commit needs to be present in the repo https://github.com/zherczeg/sljit.